content/blog/online-privacy.en.md

```diff
@@ -0,0 +1,45 @@
+{
+    "title": "Essential browser extensions for more privacy",
+    "type": "blog",
+    "date": "2019-10-21T21:11:25+00:00",
+    "description": "Installing security-focused browser extensions that block pop-up advertisements and completely encrypt website traffic from being monitored or tracked will, definitely increase your privacy—and may even speed up your browser.",
+    "featured_image": "/img/blog/online-privacy.jpg",
+    "tags": [ "privacy", "online", "adds", "cookies", "https" ],
+    "authors": [ "nishantwrp" ]
+}
+
+Websites these days collect a lot of information about you. When you leave the website, they then want to know where you are going next. They leave tracking cookies on your browser. 
+
+<img src="/img/blog/online-privacy.jpg">
+
+Tools like Google Analytics, they register everything from your geographical location to how often to visit a webpage.
+
+Below are some privacy add-ons that give your browser extra muscles to keep you safe online. It starts with ad blocker extensions that allow users to block irritating and malicious ads, save bandwidth and opt out of intrusive marketing systems.
+
```

Comment:
As mentioned in #89, we should (strongly) recommend using Firefox first. This should also include why other browsers, especially Chrome, are not a good option. There are numerous documented cases of Chrome actively working against the user's privacy interest that we could link to here.

Reply:
This also hasn't been addressed.